package.json

```diff
@@ -166,7 +166,8 @@
     ]
   },
   "dependencies": {
-    "@gnosis.pm/safe-apps-sdk": "https://github.com/gnosis/safe-apps-sdk.git#3f0689f",
+    "@gnosis.pm/safe-apps-sdk": "1.0.0-beta.4",
+    "@gnosis.pm/safe-apps-sdk-v1": "npm:@gnosis.pm/safe-apps-sdk@0.4.2",
```

Comment:
Technically this is v0, isn't it?

I suppose that when all migration finish we will only have one of these?

Reply:
Yes, v0. We will have to support v0 for some time.